spiders.html

```diff
@@ -176,10 +176,17 @@
         columnDefs: [
             {
                 targets: knownBuilds.map((_,i) => i+2),
+                type: "num",
+                className: "text-right",
                 createdCell(cell, cellData) {
                     cell.dataset.value = cellData;
                 },
-            }
+            },
+            {
+                targets: 1,
+                type: "num",
+                className: "text-right"
+            },
```

Comment:
Move this up from `columnDefs` to `columns`, there's no need for one column to appear in two places.